src/screenshot.cpp

```diff
@@ -856,3 +864,83 @@ bool MakeScreenshot(ScreenshotType t, const char *name)
 
 	return ret;
 }
+
+static byte _owner_colours[OWNER_END + 1];
+
+/**
+ * Return the colour a tile would be displayed with in the small map in mode "Owner".
+ *
+ * @param tile The tile of which we would like to get the colour.
+ * @return The colour of tile in the small map in mode "Owner"
+ */
+static inline byte GetMinimapPixels(TileIndex tile)
```

Comment:
name of this function is confusing - it only returns the colour of a single pixel. Perhaps `GetMinimapPixelColour` ?